test/Serialization/Rings.jl

```diff
@@ -0,0 +1,50 @@
+@testset "Rings" begin
+    mktempdir() do path
+        @testset "MV Polynomial Over QQ" begin
+            R, (x, y) = PolynomialRing(QQ, ["x", "y"])
+            p = x^2 + 3//4*x*y + y + 1//2
+            filename = joinpath(path, "polynomial_q.mv")
+            save(p, filename)
+            loaded = load(filename)
+            @test p == loaded
+
+        end
+
+        @testset "MV Polynomial Over ZZ" begin
+            R, (x, y) = PolynomialRing(ZZ, ["x", "y"])
+            p = x^2 + 3*x*y + y + 12
+            filename = joinpath(path, "polynomial_z.mv")
+            save(p, filename)
+            loaded = load(filename)
+            @test p == loaded
+
+        end
+
+        @testset "UV Polynomial Over ZZ Residue Ring" begin
+            R = ResidueRing(ZZ, 6)
+            MR, y = PolynomialRing(R, "y")
```

Comment:
Maybe also add a test for the MPolyRing with one variable? I.e.
```suggestion
            MR, y = PolynomialRing(R, ["y"])
```